### PR TITLE
#461 Docker image pull timeout

### PR DESCRIPTION
**Pull request by adam-sandor**
increase image pull timeout and give better error message

### CODE REVIEW

**Comment by containersol** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [The Cyclomatic Complexity of this method "pullImage" is 12 which is greater than 10 authorized.](https://github.com/ContainerSolutions/minimesos/blob/75eadd6694571e4ec388b14a954551a1263a665e/minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java#L196) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

File: minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java [hunk 1]
```diff
@@ -256,4 +229,26 @@ public static Container getContainer(String containerId) {
         return container;
     }
 
+    private static class PullImageResultCallback extends com.github.dockerjava.core.command.PullImageResultCallback {
```
Comment (containersol) [inline minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java]: MAJOR Rename this class. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2176)

**Comment by containersol** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by adam-sandor** [comment]
LGTM

**Comment by frankscholten** [comment]
LGTM


[Approved with PullApprove](https://pullapprove.com/ContainerSolutions/minimesos/pull-request/462/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ContainerSolutions/minimesos)

**Pull request merged**